python/rikai/types/geometry.py

```diff
@@ -392,6 +395,26 @@ def iou(
             return iou_arr[0]
         return iou_arr
 
+    def with_label(
+        self, text: str, color: str = get_option(CONF_RIKAI_VIZ_COLOR)
+    ):
+        """Spawn a list with current box and a `python.rikai.viz.Text`, the later one would be aligned with
+        current box on left-top, used as a convenient tool to give a box a label in visualization.
+
+        Parameters
+        ----------
+        text: str
+            The text content of that label
+        color: str
+            The color of the text, will have a default value if not given.
+
+        Returns
+        -------
+        list
+            [self, the_label]
+        """
+        return [self, Text(text, (self.xmin, self.ymin), color)]
```

Comment:
the label and the text are overlapped, is it the expected behavior

Reply:
I think for now, a hard-coded shift (shift_x, shift_y) is sufficient.

Reply:
yeah, for now just have a hard-coded padding/margin. Later on we can add additional options as needed